andela/theoden-rc
# #151980461 sell digital products

#### What does this PR do?
Allows Vendor add digital content and subsequently enables the purchase of that content

#### Description of Task to be completed?
- Enable ability to upload digital content 
- display digital content to customer
- ensure digital content skip processing and shipping and move to completed 

#### How should this be manually tested?
- Vendor can create a New product by selecting the Digital Products check box
- Go on to upload the Digital Product and Publish
- registered customer can purchase the product and receive download link when order is processed.

#### Any background context you want to provide?
only physical goods were sold.

#### What are the relevant pivotal tracker stories?
[https://www.pivotaltracker.com/story/show/151980461](https://www.pivotaltracker.com/story/show/151980461)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2017-11-05 at 7 37 41 pm" src="https://user-images.githubusercontent.com/28511567/32417948-47a6357a-c261-11e7-9066-896f00335b48.png">
<img width="1440" alt="screen shot 2017-11-05 at 7 40 43 pm" src="https://user-images.githubusercontent.com/28511567/32417950-56cef712-c261-11e7-8ace-2d109909fd33.png">
